modules/ROOT/pages/compatibility-versions-features.adoc

```diff
@@ -137,14 +137,22 @@ IMPORTANT: With xref:sdk-authentication-overview.adoc[RBAC], in Couchbase Server
 [#jdk-compat]
 == JDK Version Compatibility
 
-The Java SDK is tested with the Oracle JDK which should be used for best experience and supportability.
-Other JDK implementations might work but are  not tested and are unsupported.
+The Java SDK is tested with Oracle JDK and OpenJDK.  Other JDK implementations might work but are not tested and are unsupported.
 
-The following Oracle JDKs are supported for *Java SDK 1.4 and 2.0 - 2.7*:
+Starting with Java SDK release 2.7.2, the following JDK releases are supported:
 
-* Oracle JDK 1.6
+* OpenJDK 11 with HotSpot JVM (recommended)
+* Oracle JDK 11 (recommended)
+* OpenJDK 1.8 with HotSpot JVM
+* Oracle JDK 1.8
 * Oracle JDK 1.7
-* Oracle JDK 1.8 (recommended)
+* Oracle JDK 1.6 (Extended Support from Oracle https://www.oracle.com/technetwork/java/java-se-support-roadmap.html[ended December 2018])
```

Comment:
I thought 2.7.2 dropped support for JDK 1.6 because of the Jackson dependency?

Reply:
Oh, that's right.  Though, that's through 2.7.1.  Hm, more complicated.

Reply:
Fixed.  Note that I chose to not get into details on this page, since it's in the RNs.